Models/Gait10dof18musc/gait10dof18musc.osim

```diff
@@ -2652,56 +2607,41 @@
 						<PathPointSet>
 							<objects>
 								<PathPoint name="vas_int_l-P1">
-									<location> 0.03327 -0.220729 -0.0355644</location>
+									<location> 0.029 -0.1924 -0.031</location>
 									<body>femur_l</body>
 								</PathPoint>
 								<PathPoint name="vas_int_l-P2">
-									<location> 0.0384325 -0.239085 -0.0326963</location>
+									<location> 0.0335 -0.2084 -0.0285</location>
 									<body>femur_l</body>
 								</PathPoint>
 								<ConditionalPathPoint name="vas_int_l-P3">
-									<location> 0.0393503 -0.462338 -0.00630982</location>
+									<location> 0.0343 -0.403 -0.0055</location>
 									<body>femur_l</body>
 									<range> -2.61799 -1.42</range>
 									<coordinate>knee_angle_l</coordinate>
 								</ConditionalPathPoint>
 								<MovingPathPoint name="vas_int_l-P4">
-									<location> 0.0548266 0.025002 -0.00177934</location>
+									<location> 0.0617576 0.020984 -0.0014</location>
```

Comment:
This differs for the right side in this model, and may be the source of the symmetry differences for this exact point in all the other models too. IIRC, though, this MovingPathPoint isn't really used or am I mixing that up with another PathPoint? May be good to clean this up either way.

Reply:
Thanks @carmichaelong. I am pretty sure locations are not used with moving path points, since the location is determined by the spline (in this case). Let's talk about that during dev meeting. Thanks for the review.

Reply:
Indeed location is unused for moving path points and can be ignored.

Reply:
@carmichaelong I have set the location of the moving path points to Vec3(0) in the gait1018, gait2392, gait2354, and Hamner models. I also set the constant value of translation3 of the gait1018 models to 0.0 instead of 0.02 (not sure where this weird number came from). I had a tab issue so I replaced all spaces by tabs in one file, hope I did not mess things up. Let me know if I should do something else.

Reply:
Edits to the model look good. Let me see if I can do something about the spaces vs tabs before we merge. @antoinefalisse looks like you used 4 spaces per tab?

Reply:
Yeah, isn't that the standard?

Reply:
Yup it's the standard (and what OpenSim uses) but wanted to double check that my counting wasn't off. Note that other projects will sometimes use just 2 or 3 for economy of space though. You can see this sometimes on GitHub if you open a file with both leading spaces and tabs, they won't "line up" correctly if you use 4 spaces (i.e., below).

![image](https://user-images.githubusercontent.com/7455790/101080576-cf3c5380-355d-11eb-8881-33da1e1d5d60.png)

Reply:
Yes I had seen that bad alignment, so that's why there is one last commit to correct for that.